src/main/java/shrek/Shrek.java

```diff
@@ -0,0 +1,476 @@
+package shrek;
+
+import shrek.task.UserContent;
+import shrek.task.Deadlines;
+import shrek.task.Events;
+import shrek.task.ToDo;
+import shrek.constant.PrintStrings;
+import shrek.exception.InvalidCommandException;
+
+import java.text.ParseException;
+import java.time.DateTimeException;
+import java.util.Scanner;
+import java.util.ArrayList;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.io.File;
+import java.io.FileWriter;
+import java.time.LocalDate;
+import java.time.LocalTime;
+import java.time.format.DateTimeFormatter;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+import java.time.temporal.ChronoUnit;
+
+
+public class Shrek {
+    private static final ArrayList<UserContent> lists = new ArrayList<>();
+    private static final String LINE = PrintStrings.LINE;
+    private static final int INDEX_OF_TASK_CONTENT = 0;
+    private static final int INDEX_OF_TASK_NAME = 1;
+    private static final int INDEX_OF_TASK_COMMAND = 0;
+    private static final int INDEX_OF_TASK_INPUT = 1;
+    private static final int FIND_TASK_OR_TIME = 0;
+    private static final int FIND_CONTENT = 1;
+    private static final int INDEX_OF_TASK_MARKED = 0;
+    private static final int INDEX_OF_TIME_INPUT = 1;
+    private static final int INDEX_OF_DATE_INPUT = 0;
+    private static final int NUMBER_OF_TERMS_IN_SPLIT = 2;
+    private static final int LIST_INDEX_CORRECTION = -1;
+    public static final String NEW_LINE = System.lineSeparator();
+    private static int errorCount = 0;
```

Comment:
You might want to group/separate attributes with similar/different purpose to improve clarity and readability.